blockchain/types/block.go

```diff
@@ -121,7 +121,14 @@ func (h *Header) HashNoNonce() common.Hash {
 // Size returns the approximate memory used by all internal contents. It is used
 // to approximate and limit the memory consumption of various caches.
 func (h *Header) Size() common.StorageSize {
-	return common.StorageSize(unsafe.Sizeof(*h)) + common.StorageSize(len(h.Extra)+(h.BlockScore.BitLen()+h.Number.BitLen()+h.Time.BitLen())/8)
+	constantSize := common.StorageSize(reflect.TypeOf(Header{}).Size())
+	byteSize := common.StorageSize(len(h.Extra) + len(h.Governance) + len(h.Vote))
+	bigIntBits := common.StorageSize((h.BlockScore.BitLen() + h.Number.BitLen() + h.Time.BitLen()) / 8)
```

Comment:
Isn't it `bytes` size? since it is divided by 8. What about like `byteSizeOfBigInt`?

Reply:
`bigIntSize` also sounds good.